int8 insert
bool is_save
---
bool result